sfge/src/test/java/com/salesforce/rules/unusedmethod/SimpleCasesTest.java

```diff
@@ -61,22 +60,22 @@ public void innerInstanceMethodWithoutInvocation_expectViolation(String scope) {
      * We want tests for arity of both 0 and 1, since an explicitly declared 0-arity constructor
      * should cause a violation, unlike the implicitly generated one.
      */
+    // TODO: Enable subsequent tests as we implement functionality.
     @CsvSource({
         // One test per constructor, per visibility scope.
-        "public,  MyClass(),  0",
-        "protected,  MyClass(),  0",
-        "private,  MyClass(),  0",
-        "public,  MyClass(boolean b) , 1",
-        "protected,  MyClass(boolean b),  1",
-        "private,  MyClass(boolean b),  1"
+        //        "public MyClass(),  0",
+        //        "protected MyClass(),  0",
+        "private MyClass(),  0",
+        //        "public MyClass(boolean b) , 1",
+        //        "protected MyClass(boolean b),  1",
+        "private MyClass(boolean b),  1"
     })
-    @ParameterizedTest(name = "{displayName}: Scope {0} Method {1}, arity {2}")
-    @Disabled
+    @ParameterizedTest(name = "{displayName}: Declared constructor {0}, arity {1}")
     public void declaredConstructorWithoutInvocation_expectViolation(
-            String scope, String declaration, int arity) {
+            String declaration, int arity) {
```

Comment:
Looking at this, I realized there was no need for three arguments, so I decreased it to two.